convert2rhel/actions/pre_ponr_changes/handle_packages.py

```diff
@@ -24,10 +24,6 @@
 logger = logging.getLogger(__name__)
 
 
-def _get_pkg_names(packages):
```

Comment:
I've removed this function because in the Actions we compare these two results:
```
sorted(frozenset(pkghandler.get_pkg_nevras(pkgs_to_remove)).difference(pkgs_removed))
```
And both need to use the same building mechanism otherwise the comparism would be failing.
That is the same function for getting the nevra strings needs to be used in the Actions here and in the `remove_pkgs_unless_from_redhat()` function.

Reply:
+1